File: testing/highlight-samples/sample.go
// This is a comment
package main

import "fmt"

type User struct {
	Name string
	Age  int
}

func greet(u User) string {
	return fmt.Sprintf("Hi, %s!", u.Name)
}

var count = 42
var pi = 3.14
var active = true
var nothing = nil

File: testing/highlight-samples/sample.scala
// This is a comment
import scala.collection.mutable

class User(val name: String, val age: Int = 42) {
  val active: Boolean = true

  def greet(): String = {
    s"Hi, $name!"
  }

  /* Block comment */
}

val count = 3.14
val nothing = null
